src/megatron/energon/cli/main.py

```diff
@@ -10,7 +10,12 @@
 from megatron.energon.tools.info import command as info_command
 from megatron.energon.tools.lint import command as lint_command
 from megatron.energon.tools.mount import command as mount_command
-from megatron.energon.tools.prepare import command as prepare_command
+from megatron.energon.tools.prepare import (
+    command as prepare_command,
+)
+from megatron.energon.tools.prepare import (
+    prepare_media_command,
```

Comment:
This is not in the style of the other commands. Although I prefer not using `command as` (i.e. prefering this one). Either change all to the right name, or keep all `command`?